lib/standard_api/controller.rb

```diff
@@ -92,6 +92,26 @@ def destroy
       resources.find(params[:id]).destroy!
       head :no_content
     end
+    
+    def remove_resource
+      resource = resources.find(params[:id])
+      subresource = resource.association(params[:relationship]).klass.find_by_id(params[:resource_id])
+      
+      if(subresource)
+        result = resource.send(params[:relationship]).delete(subresource)
+      end
+      head :no_content, status: result ? :created : :bad_request
+    end
+    
+    def add_resource
+      resource = resources.find(params[:id])
+      
+      subresource = resource.association(params[:relationship]).klass.find_by_id(params[:resource_id])
+      if(subresource)
+        result = resource.send(params[:relationship]) << subresource
+      end
+      head :no_content, status: result ? :created : :bad_request
```

Comment:
I generally render the model on create. Maybe we should so here as well? Just render the subresouce. Could be persuaded not to as well.